charts/datadog/templates/NOTES.txt

```diff
@@ -333,14 +333,6 @@ On GKE Autopilot, only one "datadog" Helm chart release is allowed by Kubernetes
 * The serviceAccountName must be "datadog-agent".
 * All ConfigMap names mounted must be hardcode.
 
-{{- if eq (include "system-probe-feature" .) "true" }}
-
-#####################################################################
-####   WARNING: System Probe is not supported on GKE Autopilot   ####
-#####################################################################
-{{- fail "On GKE Autopilot environments, System Probe is not supported. The option 'datadog.securityAgent.runtime.enabled' must be set 'false'" }}
```

Comment:
Does CWS work on GKE Autopilot?

Reply:
I don't believe autopilot support has been validated by that team.

Reply:
Should there still be a block that fails if that feature is enabled then?

Reply:
Looks like there is an explicit check for CWS further down: https://github.com/DataDog/helm-charts/pull/1453/files/2bbe0fb1ce863d3bfa3f8283922f1ad210e73923#diff-5dec0964fafbdc987c2b122b6f38a16f03c7129501b7cb14fd5bbe37afb2cd0dL355